### PR TITLE
Added three Machine learning courses to Courses in English under machine learning section

### PR DESCRIPTION
**Pull request by moazamdev**
## What does this PR do?
I have added three machine learning courses

### Why is this valuable (or not)?
These courses provide in-depth knowledge of machine learning

### How do we know it's really free?
Courses are on YouTube

### For book lists, is it a book? For course lists, is it a course? etc.
These are free courses for free course list

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by moazamdev** [comment]
Hi @eshellman, I have separated my pull request as you ask to do.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -740,8 +740,11 @@
 * [Intro to Self-Driving Cars Nanodegree](https://www.udacity.com/course/intro-to-self-driving-cars--nd113) - Cezanne Camacho, Andrew Paster, Anthony Navarro, Tarin Ziyaee, et al. (Udacity)
 * [Introduction to Machine Learning](https://openlearninglibrary.mit.edu/courses/course-v1:MITx+6.036+1T2019/about) - MIT Open Learning Library
 * [Machine Learning](https://www.sololearn.com/Course/machine-learning) (SoloLearn) *(account required)*
+* [Machine Learning](https://youtube.com/playlist?list=PL7T06JEc5PF5Vuz1U7lEEaFPOExDbKVEh) - Nerd's Lesson (YouTube)
+* [Machine Learning — Andrew Ng, Stanford University](https://youtube.com/playlist?list=PLLssT5z_DsK-h9vYZkQkYNWcItqhlRJLN) - Artificial Intelligence - All in One (YouTube)
 * [Machine Learning Recipes with Josh Gordon](https://www.youtube.com/playlist?list=PLOU2XLYxmsIIuiBfYad6rFYQU_jL2ryal)
 * [Machine Learning Specialization](https://www.coursera.org/specializations/machine-learning-introduction) - Andrew Ng, Eddy Shyu, Aarti Bagul, Geoff Ladwig (Coursera)
+* [Machine Learning Tutorial in Python | Edureka](https://youtube.com/playlist?list=PL9ooVrP1hQOHUfd-g8GUpKI3hHOwM_9Dn) - Edureka (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:747]: title needs escaping

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -740,8 +740,11 @@
 * [Intro to Self-Driving Cars Nanodegree](https://www.udacity.com/course/intro-to-self-driving-cars--nd113) - Cezanne Camacho, Andrew Paster, Anthony Navarro, Tarin Ziyaee, et al. (Udacity)
 * [Introduction to Machine Learning](https://openlearninglibrary.mit.edu/courses/course-v1:MITx+6.036+1T2019/about) - MIT Open Learning Library
 * [Machine Learning](https://www.sololearn.com/Course/machine-learning) (SoloLearn) *(account required)*
+* [Machine Learning](https://youtube.com/playlist?list=PL7T06JEc5PF5Vuz1U7lEEaFPOExDbKVEh) - Nerd's Lesson (YouTube)
+* [Machine Learning — Andrew Ng, Stanford University](https://youtube.com/playlist?list=PLLssT5z_DsK-h9vYZkQkYNWcItqhlRJLN) - Artificial Intelligence - All in One (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:744]:
```suggestion
* [Machine Learning — Andrew Ng, Stanford University](https://youtube.com/playlist?list=PLLssT5z_DsK-h9vYZkQkYNWcItqhlRJLN) - Andrew Ng (YouTube)
```

File: courses/free-courses-en.md (hunk 1 above)
Comment (moazamdev) [reply courses/free-courses-en.md:747]: Hi @eshellman, Do I need to remove `| Edureka` from the title?

File: courses/free-courses-en.md (hunk 2 above)
Comment (moazamdev) [reply courses/free-courses-en.md:744]: Hi, @eshellman do I need to replace the `Artificial Intelligence - All in One` with `Andrew Ng`?

**Comment by moazamdev** [comment]
I have removed `Artificial Intelligence - All in One` with `Andrew Ng` and also removed `| Edureka` from the title

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -740,8 +740,11 @@
 * [Intro to Self-Driving Cars Nanodegree](https://www.udacity.com/course/intro-to-self-driving-cars--nd113) - Cezanne Camacho, Andrew Paster, Anthony Navarro, Tarin Ziyaee, et al. (Udacity)
 * [Introduction to Machine Learning](https://openlearninglibrary.mit.edu/courses/course-v1:MITx+6.036+1T2019/about) - MIT Open Learning Library
 * [Machine Learning](https://www.sololearn.com/Course/machine-learning) (SoloLearn) *(account required)*
+* [Machine Learning](https://youtube.com/playlist?list=PL7T06JEc5PF5Vuz1U7lEEaFPOExDbKVEh) - Nerd's Lesson (YouTube)
```
Comment (davorpa) [inline courses/free-courses-en.md:743]: Instructor name is `Pedro Domingos` and channel is `Nerd's Lesson`. How to treat this @eshellman ?

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -740,8 +740,11 @@
...
+* [Machine Learning — Andrew Ng, Stanford University](https://youtube.com/playlist?list=PLLssT5z_DsK-h9vYZkQkYNWcItqhlRJLN) - Andrew Ng (YouTube)
 * [Machine Learning Recipes with Josh Gordon](https://www.youtube.com/playlist?list=PLOU2XLYxmsIIuiBfYad6rFYQU_jL2ryal)
 * [Machine Learning Specialization](https://www.coursera.org/specializations/machine-learning-introduction) - Andrew Ng, Eddy Shyu, Aarti Bagul, Geoff Ladwig (Coursera)
+* [Machine Learning Tutorial in Python](https://youtube.com/playlist?list=PL9ooVrP1hQOHUfd-g8GUpKI3hHOwM_9Dn) - Edureka (YouTube)
```
Comment (davorpa) [inline courses/free-courses-en.md:747]: Fix tipos in creator text

```suggestion
* [Machine Learning Tutorial in Python](https://youtube.com/playlist?list=PL9ooVrP1hQOHUfd-g8GUpKI3hHOwM_9Dn) - edureka! (YouTube)
```

Comment (moazamdev) [reply courses/free-courses-en.md]: Fixed typos in creator text

File: courses/free-courses-en.md (hunk 3 above)
Comment (eshellman) [reply courses/free-courses-en.md:743]: just use `Nerd's Lesson`. The rule for youtube is to fallow the example of the resource - if the creators don't want to highlight their personal name, we shouldn't feel the need to over-rule.

Also, we are no longer adding `(YouTube)` on the courses pages - it's almost a default.

**Review by davorpa — APPROVED** [review]

File: courses/free-courses-en.md [hunk 5]
```diff
@@ -792,8 +792,11 @@
 * [Intro to Self-Driving Cars Nanodegree](https://www.udacity.com/course/intro-to-self-driving-cars--nd113) - Cezanne Camacho, Andrew Paster, Anthony Navarro, Tarin Ziyaee, et al. (Udacity)
 * [Introduction to Machine Learning](https://openlearninglibrary.mit.edu/courses/course-v1:MITx+6.036+1T2019/about) - MIT Open Learning Library
 * [Machine Learning](https://www.sololearn.com/Course/machine-learning) (SoloLearn) *(account required)*
+* [Machine Learning](https://youtube.com/playlist?list=PL7T06JEc5PF5Vuz1U7lEEaFPOExDbKVEh) - Nerd's Lesson (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:795]:
```suggestion
* [Machine Learning](https://youtube.com/playlist?list=PL7T06JEc5PF5Vuz1U7lEEaFPOExDbKVEh) - Nerd's Lesson
```

File: courses/free-courses-en.md [hunk 6]
```diff
@@ -792,8 +792,11 @@
...
+* [Machine Learning — Andrew Ng, Stanford University](https://youtube.com/playlist?list=PLLssT5z_DsK-h9vYZkQkYNWcItqhlRJLN) - Andrew Ng (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:796]:
```suggestion
* [Machine Learning — Andrew Ng, Stanford University](https://youtube.com/playlist?list=PLLssT5z_DsK-h9vYZkQkYNWcItqhlRJLN) - Andrew Ng
```

File: courses/free-courses-en.md [hunk 7]
```diff
@@ -792,8 +792,11 @@
...
 * [Machine Learning Recipes with Josh Gordon](https://www.youtube.com/playlist?list=PLOU2XLYxmsIIuiBfYad6rFYQU_jL2ryal)
 * [Machine Learning Specialization](https://www.coursera.org/specializations/machine-learning-introduction) - Andrew Ng, Eddy Shyu, Aarti Bagul, Geoff Ladwig (Coursera)
+* [Machine Learning Tutorial in Python](https://youtube.com/playlist?list=PL9ooVrP1hQOHUfd-g8GUpKI3hHOwM_9Dn) - edureka! (YouTube)
```
Comment (eshellman) [inline courses/free-courses-en.md:799]:
```suggestion
* [Machine Learning Tutorial in Python](https://youtube.com/playlist?list=PL9ooVrP1hQOHUfd-g8GUpKI3hHOwM_9Dn) - edureka!
```

**Review by eshellman — CHANGES_REQUESTED** [review]
look at other listings for syntax guidance

File: courses/free-courses-en.md (hunk 7 above)
Comment (moazamdev) [reply courses/free-courses-en.md]: Removed the `YouTube` from the listings

**Review by davorpa — APPROVED** [review]

**Review by davorpa — APPROVED** [review]

**Pull request merged**